src/hooks/useSetKeyPair.ts

```diff
@@ -7,8 +7,32 @@ import { useEffect } from "react"
 import { mutate } from "swr"
 import { useFetcherWithSign } from "utils/fetcher"
 import { recaptchaAtom, shouldUseReCAPTCHAAtom } from "utils/recaptcha"
+import { checksumAddress } from "viem"
 import useSubmit from "./useSubmit"
-import { SignProps, UseSubmitOptions } from "./useSubmit/useSubmit"
+import { MessageParams, SignProps, UseSubmitOptions } from "./useSubmit/useSubmit"
+
+function getSiweMessage({
+  addr,
+  method,
+  nonce: nonceRandom,
+  ts,
+  chainId,
+  hash,
+}: MessageParams) {
+  const nonce = `${hash}${nonceRandom}${method ?? 1}`
+
+  // Indentation is important inside the string, extra insentation would be extra whitespace in the string
```

Comment:
nit: typo 😉

Reply:
Nice catch, fixed in https://github.com/guildxyz/guild.xyz/pull/1185/commits/eed226e3f0a998c8ca87aa081124cf5ee986ac4d :ayy: